src/Symfony/Component/Serializer/Encoder/ChainDecoder.php

```diff
@@ -23,7 +23,7 @@
 class ChainDecoder implements DecoderInterface
 {
     protected $decoders = array();
-    protected $decoderByFormat = array();
+    private $decoderByFormat = array();
```

Comment:
This one is tricky... Just for protocol: I'm also not sure if this really needed.
But the field is used only in the `private function getDecoder($format)`,
so I need an advice if this needs to be reverted...